### ISSUE TITLE
What to use

### ISSUE THREAD

**Issue by ofpcarnage** [Genoil/ZECMiner#81]
Should I be using -P 0 or -P 1  or just  the default -p z ?

Intel CPU 
RX480 8gb

**Comment by zwei-und-vierzig** [comment]
You can use any options. But in you variant must be no hashrate changes.

**Comment by ofpcarnage** [comment]
what do you mean as in lower hashrate? than normal?

**Comment by jjziets** [comment]
Genoil/ZECMiner does not mine on CPU. Well that is how I understand it. you will need a CPU miner that runs on the CPU and Genoil/ZECMiner for the AMD cards.

**Comment by ofpcarnage** [comment]
Ow ok so the -P 0 or -P 1 refers to the cpu mining and not the gpu  so I'm good with just -p z

**Comment by jjziets** [comment]
No. - P is platform. If you have an integrated gpu on the cpu then it will
be platform 0 and your pcie gpus will be 1.

**Comment by jjziets** [comment]
Small -p is for pasword.

On 03 Nov 2016 11:06 AM, wrote:

No. - P is platform. If you have an integrated gpu on the cpu then it will
be platform 0 and your pcie gpus will be 1.

On 03 Nov 2016 10:23 AM, "ofpcarnage" notifications@github.com wrote:

Ow ok so the -P 0 or -P 1 refers to the cpu mining and not the gpu so I'm
good with just -p z

—
You are receiving this because you commented.
Reply to this email directly, view it on GitHub
https://github.com/Genoil/ZECMiner/issues/81#issuecomment-258084907, or mute
the thread

**Comment by ofpcarnage** [comment]
So what do I need to input for my system?
Intel Xeon Processor E3-1231 v3 (Quad Core 3.40 GHz, 8M Cache, Socket 1150 
Rx480 8gb x 6

**Comment by eds17** [comment]
You could do this, if you want to mine at zec.nanopool.org Make a batch file and put below in it...  this is for your gpu's only!

:loop
genoil.exe -f 2 -c zec-eu1.nanopool.org:6666 -u Your wallet address/Your worker name -p z
timeout /t 1 /nobreak
goto :loop

Seems to be working stable like this for me...

For your CPU maybe download Nicehash Gui miner and it will do the work for you!

**Comment by ofpcarnage** [comment]
cheers mate it works well for so long then after a while on restart it just hangs kernals wont load have to reboot the rig

**Comment by superresistant** [comment]
Loading 6 GPUs may crash your OS.

**Comment by bpvarsity** [comment]
Id recommend ethos for mining with 6 gpu's. It works great with zcash and 6x290s. Also have another with 4x380's and its great. I tried windows but it just crashes so finally just went over to ethos since they support zcash now.

**Comment by ofpcarnage** [comment]
I dont even know what ethos is I tried to google...couldn't see anything

**Comment by superresistant** [comment]
@ofpcarnage https://forum.ethereum.org/discussion/4298/ethos-linux-ethereum-mining-platform

**Comment by jjziets** [comment]
@ofpcarnage What Sol/s does that Intel Xeon Processor E3-1231 v3 (Quad Core 3.40 GHz, 8M Cache, Socket 1150 do?
